pyof/v0x01/symmetric/vendor_header.py

```diff
@@ -21,15 +21,19 @@ class VendorHeader(GenericMessage):
 
     header = Header(message_type=Type.OFPT_VENDOR)
     vendor = UBInt32()
+    additional_data = BinaryData()
```

Comment:
I would call it `data`, because the specs defines it with:

>  Vendor-defined arbitrary additional data.

Reply:
OK, I will change this.